ttnn/cpp/ttnn/operations/data_movement/untilize/device/untilize_program_factory.cpp

```diff
@@ -29,6 +30,190 @@ uint32_t get_largest_divisor(uint32_t dividend, uint32_t starting_divisor, uint3
     return 1;
 }
 
+operation::ProgramWithCallbacks untilize_multi_core_parallelize_column_subgrid(
+    const Tensor& a,
```

Comment:
is the support here 1:1 with the version without subgrids?

Asking because I'm curious if the validate needs updating.

Reply:
the regular version splits the cores into core_range and core_range_cliff but when I ran it for my shapes, the core_range_cliff was empty so I think it is 1:1.
I think validate is fine.